lib/get-image.js

```diff
@@ -1,13 +1,23 @@
-const imageSetFunctionMatchRegExp = /^(-webkit-)?image-set$/i
+const imageSetFunctionMatchRegExp = /^(-webkit-)?image-set$/i;
 
-// return a valid image
-export default node =>
+const imageFuncRegexp = /^(cross-fade|image|(repeating-)?(conic|linear|radial)-gradient|url)$/i;
+
+/**
+ * return a valid image
+ * @param {import('postcss-values-parser').ChildNode} node
+ * @returns {string|*|boolean}
+ */
+module.exports = (node) =>
 	// <url> | <image()> | <cross-fade()> | <gradient>
 	// the image-set() function can not be nested inside of itself
-	Object(node).type === 'func' && /^(cross-fade|image|(repeating-)?(conic|linear|radial)-gradient|url)$/i.test(node.value) && !(
-		node.parent.parent && node.parent.parent.type === 'func' && imageSetFunctionMatchRegExp.test(node.parent.parent.value)
+	Object(node).type === 'func' &&
+	imageFuncRegexp.test(node.name) &&
+	!(
+		node.parent.parent &&
+		node.parent.parent.type === 'func' &&
+		imageSetFunctionMatchRegExp.test(node.parent.parent.name)
 	)
-	? String(node)
-: Object(node).type === 'string'
-	? node.value
-: false;
+		? /* node.raws.before + */ String(node)
```

Comment:
if its desired to keep previous behavior of keeping new lines this should has to be uncommented

Reply:
If there’s no functional trade off to preserve spacing, I’m in favor of preserving the original formatting.

Reply:
@jonathantneal i just u;dated it